php/commands/role.php

```diff
@@ -367,6 +383,167 @@ private static function persistence_check() {
 		if ( !$wp_roles->use_db )
 			WP_CLI::error( "Role definitions are not persistent." );
 	}
+
+	private static function get_role_capability( $role_key ) {
```

Comment:
@danielbachhuber I was thinking if we can use this array of roles function in WordPress core 

i.e. : https://core.trac.wordpress.org/browser/trunk/src/wp-admin/includes/schema.php#L631

```
function populate_roles_160() {

	// Dummy gettext calls to get strings in the catalog.
	/* translators: user role */
	_x('Administrator', 'User role');
	/* translators: user role */
	_x('Editor', 'User role');
	/* translators: user role */
	_x('Author', 'User role');
	/* translators: user role */
	_x('Contributor', 'User role');
	/* translators: user role */
	_x('Subscriber', 'User role');

	$roles = array(
		'administrator' => 'Administrator',
		'editor'        => 'Editor',
		'author'        => 'Author',
		'contributor'   => 'Contributor',
		'subscriber'    => 'Subscriber'
	);
	foreach ( $roles as $role_key => $role_name ) {
		add_role( $role_key, $role_name );
		$role = get_role( $role_key );
		// We can add second parameter to fetch roles for respective versions
		$role_capabilities = get_role_capability( $role_key, "roles_160" );
		foreach ( $role_capabilities as $role_capability ) {
			$role->add_cap( $role_capability );
		}
	}
}
```
And we can use this in wp-cli, also in future for new user role we need to change/update `get_role_capability` function only. 
Let me know your thought. :)